libs/sdk-backend-tiger/src/backend/tigerSpecificFunctions.ts

```diff
@@ -786,4 +793,30 @@ export const buildTigerSpecificFunctions = (
             });
         });
     },
+    getDependentEntitiesGraph: async (workspaceId: string) => {
+        return await authApiCall(async (sdk) => {
+            return await sdk.actions
+                .getDependentEntitiesGraph({
+                    workspaceId,
+                })
+                .then((res: AxiosResponse) => {
+                    return res?.data;
+                });
+        });
+    },
+    getDependentEntitiesGraphFromEntryPoints: async (
+        workspaceId: string,
+        dependentEntitiesRequest: DependentEntitiesRequest,
+    ) => {
+        return await authApiCall(async (sdk) => {
+            return await sdk.actions
+                .getDependentEntitiesGraphFromEntryPoints({
+                    workspaceId,
+                    dependentEntitiesRequest,
+                })
+                .then((res: AxiosResponse) => {
+                    return res?.data;
+                });
```

Comment:
I would `catch` error from the promise, wrap it to `convertApiError(error)` and throw it to not expose Axios errors to the client using these methods. Some of these methods here have it, some don't as it was hard to re-write logic in the apps that used them but as these are new methods, it would be better.

Do it please for the method above as well.

Reply:
fixed.